server.go

```diff
@@ -8,174 +8,203 @@ import (
 	if nspHandler != nil {
 		nspHandler.broadcast.Join(room, connection)
 		return true
 	}
+
 	return false
 }
 
 // LeaveRoom leaves given connection from the room
 func (s *Server) LeaveRoom(namespace string, room string, connection Conn) bool {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		nspHandler.broadcast.Leave(room, connection)
 		return true
 	}
+
 	return false
 }
 
 // LeaveAllRooms leaves the given connection from all rooms
 func (s *Server) LeaveAllRooms(namespace string, connection Conn) bool {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		nspHandler.broadcast.LeaveAll(connection)
 		return true
 	}
+
 	return false
 }
 
 // ClearRoom clears the room
 func (s *Server) ClearRoom(namespace string, room string) bool {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		nspHandler.broadcast.Clear(room)
 		return true
 	}
+
 	return false
 }
 
 // BroadcastToRoom broadcasts given event & args to all the connections in the room
 func (s *Server) BroadcastToRoom(namespace string, room, event string, args ...interface{}) bool {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		nspHandler.broadcast.Send(room, event, args...)
 		return true
 	}
+
 	return false
 }
 
 // RoomLen gives number of connections in the room
 func (s *Server) RoomLen(namespace string, room string) int {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		return nspHandler.broadcast.Len(room)
 	}
+
 	return -1
 }
 
 // Rooms gives list of all the rooms
 func (s *Server) Rooms(namespace string) []string {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		return nspHandler.broadcast.Rooms(nil)
 	}
+
 	return nil
 }
 
 // Count number of connections
 func (s *Server) Count() int {
-	return s.eio.Count()
+	return s.engine.Count()
 }
 
+// ForEach sends data by DataFunc, if room does not exits sends nothing
 func (s *Server) ForEach(namespace string, room string, f EachFunc) bool {
-	nspHandler := s.getNamespace(namespace, false)
+	nspHandler := s.getNamespace(namespace)
 	if nspHandler != nil {
 		nspHandler.broadcast.ForEach(room, f)
 		return true
 	}
+
 	return false
 }
 
 func (s *Server) serveConn(c engineio.Conn) {
-	_, err := newConn(c, s.handlers)
+	err := newConn(c, s.handlers)
+
 	if err != nil {
-		root := s.handlers[""]
+		root := s.handlers[rootNamespace]
 		if root != nil && root.onError != nil {
 			root.onError(nil, err)
 		}
-		return
 	}
 }
 
-func (s *Server) getNamespace(nsp string, create bool) *namespaceHandler {
-	if nsp == "/" {
-		nsp = ""
+func (s *Server) createNameSpace(nsp string) *namespaceHandler {
```

Comment:
To keep it same everywhere `createNamespace` not `createNameSpace` :D

Reply:
ohh... sure